src/dark/process.py

```diff
@@ -31,13 +32,13 @@ def __init__(
 
     def execute(
         self,
-        command: Union[str, list[str]],
-        dryRun: Optional[bool] = None,
-        useStderr: bool = True,
-        stdout: Optional[Union[bool, TextIO, StringIO]] = False,
-        stderr: Optional[Union[bool, TextIO, StringIO]] = False,
-        **kwargs,
-    ) -> Optional[CompletedProcess]:
+        command: str | list[str],
```

Comment:
Isn't it better as `Iterable[str]`? I think you taught me that :-)

Reply:
Yes, you're right.